src/games/strategy/triplea/settings/SettingInputComponentFactory.java

```diff
@@ -139,6 +144,7 @@ private static JPanel createRadioButtonPanel(
   private static <T extends HasDefaults> SettingInputComponent<T> build(
       JPanel componentPanel,
       LabelDescription labelDescription,
+      final Optional<ValueRange> valueRange,
```

Comment:
Agreed.  This will be fixed when I apply the suggestion from your previous comment.  (I'll wait for feedback from @ron-murhammer before deciding on whether to just go with a `String` or if I'll keep `ValueRange`.)